nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/CommentManagerImpl.java

```diff
@@ -437,8 +437,7 @@ private static NuxeoPrincipal getAuthor(DocumentModel docModel) {
 
     }
 
-    public DocumentModel createLocatedComment(DocumentModel docModel, DocumentModel comment, String path)
-            {
+    public DocumentModel createLocatedComment(DocumentModel docModel, DocumentModel comment, String path) {
         try (CoreSession session = CoreInstance.openCoreSessionSystem(docModel.getRepositoryName())) {
             DocumentModel createdComment = internalCreateComment(session, docModel, comment, path);
             session.save();
```

Comment:
If you can include in you cleanup commit the remove of the defined type when you create a generic types as example `Collections.<String, Object> singletonMap` -> `Collections.singletonMap` as we did it on the 10.10.

Reply:
This code is replaced in the fix, so it's not cleaned here

Reply:
Careful, we do cleanup / format on master only.

Reply:
Dropped cleanups on 8 9 10